src/tracing/errors.test.ts

```diff
@@ -2,6 +2,11 @@ import { ErrorEvent } from '@sentry/types'
 
 import { filterKnownErrors } from './errors'
 
+Object.defineProperty(window.performance, 'getEntriesByType', {
+  writable: true,
+  value: jest.fn(),
+})
+
```

Comment:
This is expected on the DOM. Could it be put in `setupTests.ts` instead?

Reply:
we're gonna talk through this separately